src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/respository/AdjustmentRepository.kt

```diff
@@ -1,11 +1,14 @@
 package uk.gov.justice.digital.hmpps.adjustments.api.respository
 
 import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
 import org.springframework.stereotype.Repository
 import uk.gov.justice.digital.hmpps.adjustments.api.entity.Adjustment
+import java.time.LocalDate
 import java.util.UUID
 
 @Repository
 interface AdjustmentRepository : JpaRepository<Adjustment, UUID> {
-  fun findByPerson(person: String): List<Adjustment>
+  @Query("SELECT a FROM Adjustment a WHERE a.person = :person AND a.deleted = false AND (a.fromDate IS NULL OR a.fromDate >= :fromDate)")
```

Comment:
Filtering by from date is only meant to be done for UAL at the moment (And probably makes sense for RADA too). This will break a lot of the remand functionality where remand always starts before a sentence.

Reply:
Thanks @ldlharper will discuss with Steve and amend